packages/driver/cypress/e2e/commands/actions/type_special_chars.cy.js

```diff
@@ -1315,7 +1334,7 @@ describe('src/cy/commands/actions/type - #type special chars', () => {
 
       it('does not submit when keydown is defaultPrevented on input', function (done) {
         const form = this.$forms.find('#single-input').submit(() => {
-          done('err: should not have submitted')
+          done(new Error('should not have submitted'))
```

Comment:
can you explain these changes? instead of done() was not called, is this throwing an error to say we test whaat we expected?

Reply:
Good question, this just makes the error message in the reporter more intuitive! Instead of reporting "done() invoked with non-Error: err..." it just reports the error"

**Before:**
<img width="380" alt="Screen Shot 2022-08-15 at 8 43 39 AM" src="https://user-images.githubusercontent.com/26726429/184668132-e0625ddd-2765-497d-a63e-e93798af2914.png">

**After:**
<img width="399" alt="Screen Shot 2022-08-15 at 8 43 09 AM" src="https://user-images.githubusercontent.com/26726429/184668347-6b149164-123f-4889-b7a1-6f40e1327e5c.png">